sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py

```diff
@@ -1,4 +1,5 @@
 
+from typing import List
 from PIL.Image import Image
```

Comment:
what is PIL? a new library?

Reply:
PIL it's python image library. It's integrated in python